docs/user/user-guide.md

```diff
@@ -780,7 +782,8 @@ The application is available as an Alfresco AMP artifact, which is the preferred
 
 To install the AMP, follow the Alfresco AMP installation guidelines your version of Alfresco: 
 
-* [23.x](https://docs.alfresco.com/content-services/latest/install/zip/amp/)
+* [23.2 and lower](https://docs.alfresco.com/content-services/latest/install/zip/amp/)
+* [23.3 and higher](https://support.hyland.com/r/Alfresco/Alfresco-Content-Services/23.3/Alfresco-Content-Services/Install/Install-with-zip)
```

Comment:
@tl-blom-xenit The correct link is this one: [https://support.hyland.com/r/Alfresco/Alfresco-Content-Services/23.3/Alfresco-Content-Services/Install/Install-with-zip/Install-Alfresco-Module-Package](https://support.hyland.com/r/Alfresco/Alfresco-Content-Services/23.3/Alfresco-Content-Services/Install/Install-with-zip/Install-Alfresco-Module-Package).

The one you linked to is about the installation of ACS itself. The new one I gave is about the installation of an Alfresco amp. An amp is a module package that contains some extensions (like webscripts for example) that can be installed on top of ACS.